docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  twiter-node-backend:
+    build: . 
+    ports:
+      - "3000:3000"
+    links:
+      - "postgres:postgres"
+    depends_on:
+      - "postgres"
+    environment:
+      WAIT_HOSTS: postgres:5432
+      NODE_ENV: production
+  postgres:
+    image: "postgres:9.6"
+    volumes:
+      - "/var/pg-data:/var/lib/postgresql/data"
+    environment:
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
```

Comment:
Probably nothing, but sometimes, the environment variables are not populated because they don't have the same names. For example: `POSTGRES_PASSWORD: ${DB_PASSWORD}` might not get populated but `POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}` might.